pandas/tests/api/test_api.py

```diff
@@ -41,7 +41,8 @@ class TestPDApi(Base):
     ]
 
     # these are already deprecated; awaiting removal
-    deprecated_modules = []
+    # GH #28926
```

Comment:
Thanks for the thoroughness but for this change don't need the GH issue references